configs/template_outer_config.yaml

```
@@ -18,7 +18,7 @@ seed: 42
 check_stimuli_responses_match: false
 
 data:
-  root_dir: null # Choose the location of the root directory for the project
+  root_dir: ${oc.env:OPENRETINA_CACHE_DIRECTORY} # Choose the location of the root directory for the project
   data_dir: ${data.root_dir}/... # Choose the location of the data. Should be used in data_io functions.
   output_dir: ${data.root_dir}/openretina_assets/runs
```

```suggestion
  output_dir: ${hydra:runtime.output_dir}
```
for consistency with the other files